x11-wm/cwm/cwm-9999.ebuild

```diff
@@ -25,8 +25,8 @@ RDEPEND="x11-libs/libXft
 	x11-libs/libXinerama
 	x11-libs/libXrandr
 "
-DEPEND="${RDEPEND}"
-BDEPEND="sys-devel/bison
+BDEPEND="${RDEPEND}
```

Comment:
Why did you remove DEPEND?

Reply:
I thought having RDEPEND and BDEPEND should be enough yes?

Reply:
It's not. And BDEPEND is for a different architecture.
https://devmanual.gentoo.org/general-concepts/dependencies/index.html

Reply:
Ah thanks a lot. That was very informative.  
I've removed RDEPEND and replaced it with just DEPEND, which should look correct now :smile_cat: .

Reply:
Nope.
1. You need all 3. You can skip RDEPEND only if the binary package can be installed on a completely empty system without anything installed.
2. Don't mix DEPEND with BDEPEND, think of cross-compilation.

Reply:
ah, I reverted it back to what it was there previously, albeit exchanging DEPEND and RDEPEND for clarity.